htdocs/install/mysql/tables/llx_menu.sql

```diff
@@ -39,7 +39,7 @@ CREATE TABLE llx_menu
 	langs         varchar(100),								-- Lang file to load for translation
 	level         smallint,									-- Deprecated. Not used.
 	perms         text,										-- Condition to show enabled or disabled
-	enabled       text NULL default '1', 		            -- Condition to show or hide
+	enabled       text NULL, 		            -- Condition to show or hide
```

Comment:
Why removing the default value here ?

Reply:
Sorry it's an error.
I don't want to push this.
I've got an error during installation and i fix it by removing the default value.

`http://localhost/htdocs/install/step2.php`

Création de la table llx_menu et de sa clé primaire Request 200 : CREATE TABLE llx_menu ( 	rowid         integer AUTO_INCREMENT NOT NULL PRIMARY KEY, 	menu_handler  varchar(16) NOT NULL,						 	entity        integer DEFAULT 1 NOT NULL,				 	module        varchar(255),								 	type          varchar(4) NOT NULL,						 	mainmenu      varchar(100) NOT NULL,					     leftmenu      varchar(100) NULL,						 	fk_menu       integer NOT NULL,							     fk_mainmenu   varchar(100),								     fk_leftmenu   varchar(100),								 	position      integer NOT NULL,							 	url           varchar(255) NOT NULL,					 	target        varchar(100) NULL,						 	titre         varchar(255) NOT NULL,					 	langs         varchar(100),								 	level         smallint,									 	perms         text,										 	enabled       text NULL default '1', 		             	usertype      integer NOT NULL DEFAULT 0,				 	tms           timestamp 	 ) ENGINE=innodb; Executed query : CREATE TABLE llx_menu ( 	rowid         integer AUTO_INCREMENT NOT NULL PRIMARY KEY, 	menu_handler  varchar(16) NOT NULL,						 	entity        integer DEFAULT 1 NOT NULL,				 	module        varchar(255),								 	type          varchar(4) NOT NULL,						 	mainmenu      varchar(100) NOT NULL,					     leftmenu      varchar(100) NULL,						 	fk_menu       integer NOT NULL,							     fk_mainmenu   varchar(100),								     fk_leftmenu   varchar(100),								 	position      integer NOT NULL,							 	url           varchar(255) NOT NULL,					 	target        varchar(100) NULL,						 	titre         varchar(255) NOT NULL,					 	langs         varchar(100),								 	level         smallint,									 	perms         text,										 	enabled       text NULL default '1', 		             	usertype      integer NOT NULL DEFAULT 0,				 	tms           timestamp 	 ) ENGINE=innodb; | Erreur SQL DB_ERROR_1101 BLOB, TEXT, GEOMETRY or JSON column 'enabled' can't have a default value
-- | --

Reply:
You are finally right. Because it has been modified into "text" (previously varchar), we can't keep a default value. I fixed it with this commit: c7ca99accb2db6ca1a1f8a47d84f0c87dadff54e